static/app/components/core/button/styles.tsx

```diff
@@ -211,7 +209,7 @@ export function DO_NOT_USE_getButtonStyles(
     css`
       height: auto;
       min-height: auto;
-      padding: ${space(0.25)};
+      padding: ${p.theme.space['2xs']};
```

Comment:
This is unfortunate that we'll have to be inconsistent with dotted vs bracketed attribute access

Reply:
yeah we talked about this too. @Jesse-Box favours `xxs` and `xxl` over `2xl` and that would solve this problem too.